packages/alea-frontend/pages/course-view/[courseId].tsx

```diff
@@ -404,6 +405,7 @@ const CourseViewPage: NextPage = () => {
                   </Box>
                 ),
               }}
+              currentSlideNum={slideNum}
```

Comment:
No need to pass `currentSlideNum` and `selectedSectionTOC`. Just set the right uri